### PR TITLE
avatar and badge components with doc added

### PR DESCRIPTION
**Pull request by devdeadviz**

### CODE REVIEW

File: pages/components/avatar/avatar.html [hunk 1]
```diff
@@ -0,0 +1,271 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
```
Comment (Mohasin-Haque) [inline pages/components/avatar/avatar.html]: This is not the right for mate to write meta tags.. The information is passed in contents so.. You cans is :-

<meta name="author" content="Kuldeep">
<meta name="description" content="about you uu">

File: pages/components/avatar/avatar.html [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+          <ul>
+            <li>
+              <a
+                target="_blank"
+                class="social-icons-wrapper"
+                href="https://twitter.com/devdeadviz"
+                ><i class="fab fa-twitter"></i
+              ></a>
+            </li>
+            <li>
```
Comment (Mohasin-Haque) [inline pages/components/avatar/avatar.html:29]: Please run prettier once more to get to the indentation

File: pages/components/avatar/avatar.html [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+          <a class="drawer-lists" href="../image/image.html">
+            <p class="drawer-list-content">Image</p>
+          </a>
+          <a class="drawer-lists" href="../input/input.html">
+            <p class="drawer-list-content">Input</p>
+          </a>
+        </div>
+      </aside>
+      <main class="main-content m-5 width-90">
+        <h1 class="mb-4">Avatar</h1>
```
Comment (Mohasin-Haque) [inline pages/components/avatar/avatar.html:97]: MB4???

File: pages/components/badge/badge.css [hunk 4]
```diff
@@ -0,0 +1,76 @@
+@import "../../../styles/typography.css";
```
Comment (Mohasin-Haque) [inline pages/components/badge/badge.css:2]: can put comment of external import for sanity.

**Review by Mohasin-Haque — CHANGES_REQUESTED** [review]
Your code looks good but , Please go through the requested changes.. It needs some sanity.. And you are good to go😊

**Review by saifudinbadani — APPROVED** [review]
Looks good to merge.

File: pages/components/avatar/avatar.html (hunk 3 above)
Comment (devdeadviz) [reply pages/components/avatar/avatar.html:97]: The standard way of representing margins. Here mb-4 means margin-bottom: calc(4*var(--margin)).

File: pages/components/avatar/avatar.html (hunk 1 above)
Comment (devdeadviz) [reply pages/components/avatar/avatar.html]: I will correct this. Thanks for the input.

File: pages/components/badge/badge.css (hunk 4 above)
Comment (devdeadviz) [reply pages/components/badge/badge.css:2]: Comments added.

**Review by Mohasin-Haque — APPROVED** [review]
Now it looks good.. Please proceed!!. 😊

**Pull request merged**